hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/SchemaEvolutionContext.java

```diff
@@ -82,32 +81,25 @@ public class SchemaEvolutionContext {
 
   private final InputSplit split;
   private final JobConf job;
-  private HoodieTableMetaClient metaClient;
+  private final HoodieTableMetaClient metaClient;
   public Option<InternalSchema> internalSchemaOption;
 
-  public SchemaEvolutionContext(InputSplit split, JobConf job) throws IOException {
+  public SchemaEvolutionContext(InputSplit split, JobConf job) {
     this(split, job, Option.empty());
   }
 
-  public SchemaEvolutionContext(InputSplit split, JobConf job, Option<HoodieTableMetaClient> metaClientOption) throws IOException {
+  public SchemaEvolutionContext(InputSplit split, JobConf job, Option<HoodieTableMetaClient> metaClientOption) {
     this.split = split;
     this.job = job;
     this.metaClient = metaClientOption.isPresent() ? metaClientOption.get() : setUpHoodieTableMetaClient();

```

Comment:
now the init of internalSchemaOption has been removed.
https://github.com/apache/hudi/blob/83d4fe15b4f50a58e919390217b71bb95e952085/hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/AbstractRealtimeRecordReader.java#L91  need to modify

Reply:
oh ok then there is no need of this PR.